frontend/front/src/pages/Admin/home/HomeTable.js

```diff
@@ -99,32 +122,61 @@ const HomeTable = () => {
   ];
 
   const {
-    data: homesData,
-    isError: isHomesError,
+    data: fetchedHomesData,
+    isError: isFetchedHomesError,
     isLoading: isHomesDataLoading,
   } = useGetHomesQuery();
+  const {
+    data: surveyVisitsData,
+    isError: isSurveyVisitsError,
+    isLoading: isSurveyVisitsDataLoading,
+  } = useGetSurveyVisitsQuery();
+
+  let homesData = [];
 
-  if (isHomesDataLoading) {
+  if (surveyVisitsData && fetchedHomesData) {
+    homesData = Object.values(fetchedHomesData).map((home) => {
+      const visit = surveyVisitsData.find((visit) => visit.home_id === home.id);
+      if (visit) {
+        return { ...home, completed: true, survey_id: visit.id };
```

Comment:
`home`s should have `completed` on them by default on `main` now, so you don't need to set it here. Also, for the record, I think that API document is out of date because the endpoint `/survey_visits?home_id={id}` totally works. It's still probably more appropriate here to get the whole survey_visit list since they have to be matched to the entire list of homes. That, or you could try adding `:survey_visit_ids` to the `json.extract!` line in `_home.json.jbuilder` in the back end code to have it return that data..

Reply:
This is no longer there. 

Thanks for letting me now, I checked in postman. I'll just do that from now on.